about.html

```diff
@@ -30,32 +37,33 @@
                 </g>
               </svg>
             </a>
-            <div>
-                <ul class="flex nav text-dark">
-                    <li><a href="./about.html">About</a></li>
-                    <li><a href="#">Program</a></li>
-                    <li><a href="#">Join</a></li>
+            <div class="navbar flex">
+                <ul class="flex nav">
+                    <li><a class="link" href="./about.html">About</a></li>
+                    <li><a class="link" href="#">Program</a></li>
+                    <li><a class="link" href="#">Join</a></li>
                 </ul>
+                <button class="btn-campaign" type="button"> CC Campaign    </button>
             </div>
-        <button class="btn-campaign" type="button"> CC Campaign    </button>
     </nav>
       
     <h1 class="mg-2 center">
```

Comment:
- [ ] The background dos not respect the accessibility rules and is not the same as the provided design. please consider changing the background image or adding an overlay with opacity on top of the current image, this will make the design closer to the CC design

|Your design|CC design|
|-|-|
|![Screenshot_20220528_060409](https://user-images.githubusercontent.com/25227022/170810982-f4aa0fd7-ad06-4285-8186-b2a38d4298b0.png)|![Screenshot_20220528_060725](https://user-images.githubusercontent.com/25227022/170811002-f8e4949a-ddfc-4f9a-a84b-a57d6ac7e855.png)|